scripts/lib/mkcloud-libvirt.sh

```diff
@@ -103,6 +103,12 @@ function libvirt_prepare()
 
 function libvirt_setupadmin()
 {
-    ${mkcloud_lib_dir}/libvirt/admin-config $cloud $admin_node_memory $adminvcpus $emulator $admin_node_disk "$localreposdir_src" "$localreposdir_target" > /tmp/$cloud-admin.xml
+    nvramtemplate=""
+    if [[ $(uname -m) == x86_64 ]] && [[ "$firmware_type" == "uefi" ]]; then
+        nvramtemplate="/usr/share/qemu/ovmf-x86_64-ms-vars.bin"
+        cp $nvramtemplate /var/lib/libvirt/qemu/nvram/cloud-admin_VARS.fd
```

Comment:
@dirkmueller: libvirt can take care of this only if the `nvram` is explicitly specified in qemu.conf which is not done currently. If not, we need to provide the store as a template (as above). I get the below error if neither of this is done. AFAIU, the template can override the conf. Which do you think is a better option ?

```
    dom.create()
  File "/usr/lib64/python2.7/site-packages/libvirt.py", line 1035, in create
    if ret == -1: raise libvirtError ('virDomainCreate() failed', dom=self)
libvirt.libvirtError: operation failed: unable to find any master var store for loader: /usr/share/qemu/ovmf-x86_64-ms-code.bin
```